arrow-core/src/main/kotlin/arrow/core/extensions/setk.kt

```diff
@@ -109,3 +141,108 @@ interface SetKEqK : EqK<ForSetK> {
       }
     }
 }
+
+/**
+ * [Semigroup] instance definition for [SetK]
+ */
+fun <A> Semigroup.Companion.setK(): Semigroup<SetK<A>> = Semigroup<SetK<A>> { b ->
```

Comment:
All these should be defined for `Set` and not for `SetK`.
They should be defined in `arrow-core-data` in the `arrow.core` package.